install.sh

```diff
@@ -4,7 +4,7 @@
 user="$SUDO_USER"
 [[ -z "$user" ]] && user="$(id -un)"
 home="$(eval echo ~$user)"
-readonly RP_SETUP_DIR="$home/RetroPie-Setup"
+readonly RP_SETUP_DIR=$(locate -b RetroPie-Setup)
```

Comment:
if your setup is not in home, maybe an alternative would be to allow for a `.gitignore` file that the user can add local values ... you can also add an example `local.conf.example`

In there, you can define a `$path_to_setup` and the above `install.sh` would set $home (or other, not sure where else hone is used) to either that local defined value _or_ its default `"$(eval echo ~$user)"` ... 

Id run this idea past @meleu before starting in the event he doesnt like it .. but allowing a local overide in general could add nice flexibility to this script going forward

Reply:
that's the situation, my installation uses dot ".Retropie-Setup" and the script does not work. another option would be to search for retropie_setup.sh, make it more specific, and maybe even ask which place is right with a list

Reply:
This week maybe I will try a new path and update this PR.